src/core/utils/index.js

```diff
@@ -233,7 +232,7 @@ export function addonHasVersionHistory(addon) {
     throw new Error('addon is required');
   }
 
-  return ![ADDON_TYPE_OPENSEARCH].includes(addon.type);
```

Comment:
So can you remove `addonHasVersionHistory()` entirely now?

Reply:
Yes, I was able remove addonHasVersionHistory() from the code base.

Reply:
I still see it in this file, though :)

Reply:
Oops, let me check again.

Reply:
Sorry, It's still there. Removed it now.